src/ExecuteTimeWidget.ts

```diff
@@ -324,6 +327,13 @@ export default class ExecuteTimeWidget extends Widget {
             msg += ` at ${getTimeString(endTime, this._settings.dateFormat)}`;
           }
           msg += ` in ${executionTime}`;
+
+          const numberOfOutputs = cell.model.outputs.length;
+          if (this._settings.showOutputsPerSecond && numberOfOutputs > 0) {
+            const outputsPerSecond = executionsPerSecond / numberOfOutputs;
+            msg += ` and generated ${numberOfOutputs} output(s)`;
```

Comment:
This is getting a bit long:

![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/af41e0a5-5f84-49f3-816a-98bb04e28e29)

Can you modify this line to only append "s" when the number of outputs is >1? This will also remove the parentheses making it shorter in both cases.

additional idea: "and generated" could be replaced by shorter "displaying"

Reply:
Makes sense. It looks like this now. Much shorter 👍

![image](https://github.com/deshaw/jupyterlab-execute-time/assets/970514/b3cdc770-d3de-4923-98bf-1e868fd00eb8)

Reply:
@krassowski I am not familiar with ui-tests. How can we get something like last-executed.png?

Reply:
It will be auto-generated when running the test for the  first time: https://github.com/deshaw/jupyterlab-execute-time/tree/master/ui-tests#run-the-tests

Reply:
But in general I would just push the code and download the artifacts from CI